app/models/page_snapshot.rb

```diff
@@ -12,7 +12,16 @@ def document
   end
 
   def match_text
-    document.css(self.page.css_selector).text
+    @match = document.css(self.css_selector)
```

Comment:
Throughout this method you refer to properties on the `Page` as if they existed on the `PageSnapshot`. This should be `self.page.css_selector`. Similarly with the `exclude_selector` below.

Reply:
Thanks @GabeIsman! Not sure how I missed that.